### ISSUE TITLE
Allow zoomInterval smaller than 0.5

### ISSUE THREAD

**Issue by megatom** [Leaflet/Leaflet.Graticule#4]
If i use a zoomInterval smaller than 0.5 the labels getting weird.
With 0.2 for example i get a label with 50.800000000000000001W instead of 50.8W.
Same for 0.1.

**Comment by srappel** [comment]
Yes, I had to write some crazy modifications to the calculations before I could convert the labels to DMS.  You can see my sloppy code on my Geodex repo.  It's right in the graticule js file in the plugins folder.

**Comment by megatom** [comment]
I did a quick and dirty workaround for me, because i also wanted to have negative north values instaed of positive south values:

if(lat % 1 === 0) return '' + lat + 'N';
else return '' + lat.toFixed(2) + 'N';

This works for intervall 0.01 and bigger and i am fine with this.

Same for Longitude:
if(lng % 1 !== 0) lng = lng.toFixed(2);

**Comment by rasamimalala** [comment]
I already solve this issue in my pull request: https://github.com/Leaflet/Leaflet.Graticule/pull/5

**Comment by megatom** [comment]
How can I access the complete changed file now?

**Comment by rasamimalala** [comment]
My pull request is not yet accepted. You can find a fork which containing my modification by following this link: https://github.com/tagip/Leaflet.Graticule
